menu-definitions.lisp

```diff
@@ -216,6 +226,12 @@ unmark the entry at the selected point."
   "Default action is to do nothing"
   (declare (ignore key-seq)))
 
+(defun match-p (table-item menu)
+  (funcall (single-menu-filter-pred menu)
+           (car table-item)
+           (second table-item)
+           (single-menu-current-input menu)))
+
```

Comment:
Why did you pull this out of the labels form? Please don't break methods that are internal to other methods. It pollutes the local namespace.